tools-local/targetgroups.props

```diff
@@ -189,33 +189,46 @@
       <Imports>netcoreapp2.0</Imports>
       <CompatibleWith>netstandard</CompatibleWith>
     </TargetGroups>
-    <!-- netcoreapp is an alias for netcoreapp2.1 -->
-    <TargetGroups Include="netcoreapp">
-      <TargetFramework>netcoreapp2.1</TargetFramework>
-      <NuGetTargetMoniker>.NETCoreApp,Version=v2.1</NuGetTargetMoniker>
-      <NuGetTargetMonikerShort>netcoreapp2.1</NuGetTargetMonikerShort>
+    <TargetGroups Include="netcoreapp2.2">
+      <TargetFramework>netcoreapp2.2</TargetFramework>
+      <NuGetTargetMoniker>.NETCoreApp,Version=v2.2</NuGetTargetMoniker>
+      <NuGetTargetMonikerShort>netcoreapp2.2</NuGetTargetMonikerShort>
       <Imports>netcoreapp2.1</Imports>
       <CompatibleWith>netstandard</CompatibleWith>
     </TargetGroups>
-    <TargetGroups Include="netcoreapp1.2">
-      <TargetFramework>netcoreapp1.2</TargetFramework>
-      <NuGetTargetMoniker>.NETCoreApp,Version=v1.2</NuGetTargetMoniker>
-      <NuGetTargetMonikerShort>netcoreapp1.2</NuGetTargetMonikerShort>
+    <TargetGroups Include="netcoreapp3.0">
+      <NETCoreAppMaximumVersion>3.0</NETCoreAppMaximumVersion>
+      <TargetFramework>netcoreapp3.0</TargetFramework>
+      <NuGetTargetMoniker>.NETCoreApp,Version=v3.0</NuGetTargetMoniker>
+      <NuGetTargetMonikerShort>netcoreapp3.0</NuGetTargetMonikerShort>
+      <Imports>netcoreapp2.2</Imports>
+      <CompatibleWith>netstandard</CompatibleWith>
     </TargetGroups>
-    <TargetGroups Include="netcoreapp2.1aot">
-      <TargetFramework>netcoreapp2.1</TargetFramework>
+    <!-- netcoreapp is an alias for netcoreapp3.0 -->
+    <TargetGroups Include="netcoreapp">
```

Comment:
We probably don't want to also do the netstandard mapping at this point?

Reply:
There is no new netstandard beyond 2.0 yet so everything still maps to that.